### PR TITLE
Rig2

### PR DESCRIPTION
**Pull request by leoscholl**
Changes to make rig2 work

### CODE REVIEW

**Comment by leoscholl** [comment]
Things I've changed:
- make a rig settings file in config/rig_defaults.py to save settings on a rig-specific manner
- add a "rig_name" column to the database
- change the rig name from a single setting to just use the hostname of the machine running bmi3d instead
- fix the web interface so the rig_name column is included
- add a button to the web interface to show experiments from other rigs (by default only the local rig is shown)

**Comment by leoscholl** [comment]
@katherineperks this is ready for review!

**Review by resatomo — APPROVED** [review]

File: config/rig_defaults.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    window_size = (2560, 1440)
+    screen_dist = 28
+    screen_half_height = 10.75
+    default_db = 'rig1'
+elif hostname == 'siberut-bmi':
+    optitrack_address = '10.155.204.10'
+    optitrack_save_path = "C:/Users/aolab/Documents",
+    screen_dist = 28
+    screen_half_height = 10.25
+    default_db = 'rig2'
```
[the remark this answers is not in the document]
Comment (katherineperks) [reply config/rig_defaults.py:110]: Update - we are keeping the main booth db called "rig1"
Shouldn't the default db of the siberut booth be changed to rig1?

**Review by katherineperks — APPROVED** [review]
Everything looks good to me. I left one comment about the default db we want to use for the siburet booth, but this can be changed after merging since no one will immediately be using siburet

**Pull request merged**